modules/ROOT/pages/migration-from-7x.adoc

```diff
@@ -3,9 +3,628 @@
 :description: Guidance for migrating from TinyMCE 7 to TinyMCE 8
 :keywords: migration, considerations, premigration, pre-migration
 :release-version: 8.0
+:page-toclevels: 3
 
-// TODO
+== Introduction
 
-== {productname} {release-version} core changes.
+This guide provides a comprehensive overview of the breaking changes introduced in {productname} {release-version}, along with the necessary steps to migrate from {productname} 7.x. It covers key updates to APIs, plugins, and service configurations, including deprecated methods, renamed components, and removed features. These changes are designed to enhance performance, simplify configuration, and align with modern web standards, ensuring a smoother transition and continued compatibility for your integrations.
 
-For additional details on {productname} {release-version} changes, see xref:8.0-release-notes.adoc#overview[{productname} {release-version} release notes].
\ No newline at end of file
+[IMPORTANT]
+.Breaking Changes Quick Reference
+====
+The following table summarizes all breaking changes in {productname} {release-version}. Each item links to detailed information further in the guide.
+
+Any items marked **"High"** level require immediate attention during migration.
+
+[cols="2,3,1",options="header"]
+|===
+|Breaking Change |Impact |Level
+|xref:license-key-system-update[License Key System]
+|Self-hosted deployments now require a new license key format and license key manager. Old keys are **not compatible**.
+|High
+
+|xref:dompurify-update-breaking-change[DOMPurify Update]
+|Sanitization is now stricter; content previously allowed may be stripped or altered.
+|High
+
+|xref:editor-selection-setcontent-deprecated[editor.selection.setContent]
+|Method deprecated. Use `editor.insertContent` instead.
+|Medium
+
+|xref:fire-method-deprecation[fire()]
+|Method deprecated. Use `dispatch()` for event handling.
+|Medium
+
+|xref:pagebreak-split-block-default[pagebreak_split_block]
+|The default value changed from `false` to `true`, affecting how page breaks interact with block elements.
+|Low
```

Comment:
I think this should be `medium`, and also see my comments about page break below